server/src/main/kotlin/core/gameflow/showdown.kt

```diff
@@ -4,3 +4,42 @@ sealed class ShowdownAction
 
 data class ShowCards(val playerId: Int) : ShowdownAction()
 data class MuckCards(val playerId: Int) : ShowdownAction()
+
+fun resolveShowdown(handState: HandState): List<ShowdownAction> {
+
+    val orderedPlayers = when {
+        handState.lastAggressor == null -> handState.orderedPlayers(handState.buttonPosition + 1)
+        else -> handState.orderedPlayers(handState.lastAggressor.position)
+    }.filter { it.isInGame }
+
+    // when only one player is left in the game
+    if (orderedPlayers.size == 1)
+        return listOf(MuckCards(orderedPlayers.first().id))
+
+    // all players must show their cards if there are more streets to be dealt
```

Comment:
I guess this is 'All in-game-players are All-In'-case?

Reply:
precisely

Reply:
ok